### DISCUSSION TITLE
Portal Themes

### DISCUSSION THREAD

**Discussion by cmdigitaluk** [openstore-ecommerce/OpenStore#270] [Q&A] [answered]
I've copied the Themes folder from Desktop Modules to the portal level. However the only way I can seem to get my portal theme to be recognised is to edit all the @RenderTemplate file locations in the portal files so they link to the portal directory not DesktopModules.  Am I doing something wrong?

**Comment by skamphuis** [comment]
That's correct. The intended use (but @leedavi can correct me if I'm wrong) is to only copy over the files you need to change, in stead of all. In most cases that will only be a handful (or 2 hands) of files.

Hope this helps.

**Comment by leedavi** [comment, marked as answer]
If you want a completely new theme you can copy it to "\DesktopModules\NBright\NBrightBuy\Themes\<newtheme>" (system level)  but you will need to change all the paths for the @RenderTemplate tokens.

The idea of using the portal is that an upgrade of OpenStore does not overwrite your template changes at the system level.

Basically if you need to change a template, move it to the portal level.  If you don't need to change it, no need to move it.  If you create your owe complete theme, no need to move it to portal level, leave it at system level.  An upgrading will not have your theme, so no risk of overwrite.

**Comment by cmdigitaluk** [comment]
That's great thank you.  Glad to know I wasn't doing it completely wrong.
